dags/dim_active_users_aggregates.py

```diff
@@ -40,7 +40,7 @@
 Config = namedtuple("Config", "exec_date apps")
 CONFIGS = {
     "desktop": Config("{{ ds }}", ("firefox_desktop",)),
-    # "mobile": Config("{{ macros.ds_add(ds, -1) }}", ("firefox_ios", "fenix",)),
+    "mobile": Config("{{ macros.ds_add(ds, -1) }}", ("fenix", "focus_android", "firefox_ios", "focus_ios",)),
```

Comment:
We also have [klar iOS in the aggregates](https://github.com/mozilla/bigquery-etl/blob/b9f0ff69b4c529a31f3ed78f2f5a2bd41eb378b1/sql_generators/active_users/__init__.py#L22-L26). Can we please add it for the mobile totals?

Reply:
Added.